docs/topics/auth/default.txt

```diff
@@ -762,8 +762,10 @@ Redirecting unauthorized requests in class-based views
 ------------------------------------------------------
 
 To ease the handling of access restrictions in :doc:`class-based views
-</ref/class-based-views/index>`, the ``AccessMixin`` can be used to redirect a
-user to the login page or issue an HTTP 403 Forbidden response.
+</ref/class-based-views/index>`, the ``AccessMixin`` can be used to configure
+the behavior of a view when the conditions are not met.  An authenticated user
```

Comment:
"when the conditions are not met" sounds just a bit too vague to me. How about "when the request is denied" or "when access is denied"?

Reply:
👍